src/modules/job-manager/job.h

```diff
@@ -40,6 +40,7 @@ struct job {
     uint8_t free_pending:1; // free request sent to sched
```

Comment:
In the description, could you indicate that a boolean flag was added to the start request payload?  it's kind of vague with the current wording.  Also, it's for any job that's in the RUN state per its replayed event log, not a "belief" per se (you're giving job manager too much credit!)